### ISSUE TITLE
Error Uploading File

### ISSUE THREAD

**Issue by wyrl** [google/php-photoslibrary#19]
Type: Error

Message: Call to undefined method Google\Photos\Library\V1\NewMediaItem::setFileName()

Filename: ..vendor/google/photos-library/src/Google/Photos/Library/V1/PhotosLibraryResourceFactory.php

Line Number: 100

**Comment by jfschmakeit** [comment]
Thanks, we have been able to reproduce this as well and are preparing a change to fix this. (The current workaround is to create a `NewMediaItem` and `SimpleMediaItem` directly): 

```
$simpleMediaItem = new SimpleMediaItem();
$simpleMediaItem->setUploadToken($uploadToken);
$simpleMediaItem->setFileName($fileName);

$newMediaItem = new NewMediaItem();
$newMediaItem->setSimpleMediaItem($simpleMediaItem);
```

**Comment by jfschmakeit** [comment]
This has been fixed in commit a8b73e0 and released in release  `v1.6.1`.
